File: ASSIGNMENT 2(Java)/greaterof2numbers.java
import java.util.Scanner;
class Greater
{
    public static void main(String args[])
    {
        int num1,num2;
        System.out.println("Enter First Number : ");
        Scanner sc = new Scanner(System.in);
        num1 = sc.nextInt();
        System.out.println("Enter Second Number : ");
        num2 = sc.nextInt();	
if(num1>num2)
        {
            System.out.println("The Number1 is Greater");
        }
      else
        {
            System.out.println("The Number2 is Greater");
        }
    }
}
 

File: MACHINE RUN 11-11-2020/check greatest of 2 numbers/greater of 2 numbers.c
#include<iostream>
using namespace std;
class Greater
{
    int num1,num2;
    public:
        void Check_Greater()
        {
            cout<<"Enter two numbers";
            cin>>num1>>num2;
            if(num1>num2)
            {
                cout<<"num1 is greater";
            }
        else
            {
                cout<<"num2 is greater";

            }
        }
};

int main()
{
    Greater g1;
    g1.Check_Greater();
}


